### ISSUE TITLE
validation.condition of additional_thumbprints has not been completed (v1.3.0)

### ISSUE THREAD

**Issue by officel** [unfunco/terraform-aws-oidc-github#25]
If additional_thumbprints is not required, it should not be an error to not include it in the code.

```
module "aws_oidc_github" {
  source = "unfunco/oidc-github/aws"
<ommit>
  # no additional_thumbprints variable
}

```

has error

```
│ Error: Invalid function argument
│
│   on .terraform/modules/aws_oidc_github/variables.tf line 21, in variable "additional_thumbprints":
│   21:     condition     = length(var.additional_thumbprints) <= 4
│     ├────────────────
│     │ while calling length(value)
│     │ var.additional_thumbprints is null
│
│ Invalid value for "value" parameter: argument must not be null.
```

workaround(and If additional_thumbprints is required, then) 

```
module "aws_oidc_github" {
  source = "unfunco/oidc-github/aws"
<ommit>

  additional_thumbprints = []

}
```

Additional checks on condition would be nice.

like https://stackoverflow.com/questions/66912991/terraform-custom-validation-for-variable-that-can-be-null

**Comment by Antvirf** [comment]
Ran into this just now and submitted PR https://github.com/unfunco/terraform-aws-oidc-github/pull/26

**Comment by unfunco** [comment]
Fixed in #26 – thanks both! A new release will be published shortly.
